components/binance/browser/binance_service_browsertest.cc

```diff
@@ -736,6 +744,13 @@ IN_PROC_BROWSER_TEST_F(BinanceAPIBrowserTest, RevokeToken) {
   ResetHTTPSServer(base::BindRepeating(&HandleRequest));
   EXPECT_TRUE(NavigateToNewTabUntilLoadStop());
   auto* service = GetBinanceService();
+
+  // Set an access token
+  ASSERT_TRUE(service->GetAccessToken("abc123",
```

Comment:
We know that tokens were set when this assertion is true: https://github.com/brave/brave-core/blob/master/components/binance/browser/binance_service.cc#L188